setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='batavia',
-    version='3.4.0.dev1',
+    version='3.5.0dev1',
```

Comment:
this was the only significant merge conflict. Im not quite sure what is an expected value here now since this patch allows to run both Py3.4 and Py3.5